src/hooks/client/ApiQueryClient.ts

```
@@ -157,8 +157,26 @@ export class ApiQueryClient extends ApiBaseClient {
       phone_number: formData.coordinatorPhoneNumber,
     };
     const coordinatorResponse = await this.axiosInstance.post(`/trl/coordinator`, coordinatorPayload);
+    // 0. Handle Case File Uploads via Signed URL
+    let casesAttachments: string[] = [];
+    if (formData.researchDetailsFiles && formData.researchDetailsFiles.length > 0) {
+      console.log('📎 Uploading files for case...');
+      for (const file of formData.researchDetailsFiles) {
+        console.log(`Processing file: ${file.name}, type: ${file.type}, size: ${file.size}`);
+        try {
+          const { upload_url, object_path } = await this.usePresignUpload(file);
```

Biome (2.3.13)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)